chef/cookbooks/neutron/recipes/common_agent.rb

```diff
@@ -49,19 +49,20 @@
   subscribes :run, resources(:cookbook_file=> disable_rp_filter_file), :delayed
 end
 
+if neutron[:neutron][:networking_plugin] == "ml2"
+  ml2_mech_drivers = neutron[:neutron][:ml2_mechanism_drivers]
+  case
+  when ml2_mech_drivers.include?("openvswitch")
+    neutron_agent = node[:neutron][:platform][:ovs_agent_name]
+    neutron_agent_pkg = node[:neutron][:platform][:ovs_agent_pkg]
+    agent_config_path = "/etc/neutron/plugins/openvswitch/ovs_neutron_plugin.ini"
+  when ml2_mech_drivers.include?("linuxbridge")
+    neutron_agent = node[:neutron][:platform][:lb_agent_name]
+    neutron_agent_pkg = node[:neutron][:platform][:lb_agent_pkg]
+    agent_config_path = "/etc/neutron/plugins/linuxbridge/linuxbridge_conf.ini"
+  end
```

Comment:
> So if we specify mech_drivers = ["openvswitch", "linuxbridge"], when will the linuxbridge agent ever be installed?

You can't use both in parallel. See https://github.com/toabctl/barclamp-neutron/commit/a66c76cac342de85055734064d93c86b1eff5542#diff-1cab7f3b6842279ae4d6bd875e073cacR114